specs/PermissionManagement.md

```diff
@@ -503,6 +509,12 @@ namespace Microsoft.Web.WebView2.Core
         // developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
         // to query the system fonts available for styling web content.
         LocalFonts = 10,
+
+        // Indicates permission to send and receive system exclusive messages to/from MIDI
+        // (Musical Instrument Digital Interface) devices. Permission is requested
+        // when developers use the [Web MIDI API](https://developer.mozilla.org/en-US/docs/Web/API/Web_MIDI_API)
+        // to request access to system exclusive MIDI messages.
+        MidiSystemExclusiveMessageAccess = 11,
```

Comment:
@david-risney sorry to come back again, but I think an even better name would be `MidiSystemExclusiveMessages`. The "Access" doesn't belong here -- none of the other permissions have "Access" in the name. And it's clearer that what the site has permission to is "MIDI system exclusive messages", not any sort of access.

Reply:
Thats true. These enums are all kinds of permissions and having 'access' or 'permission' or similar in the name is redundant.

@vickiez can we please update this